chalicelib/checks/deployment_checks.py

```diff
@@ -45,6 +47,22 @@ def cleanup_tempdir(tempdir):
     shutil.rmtree(tempdir)
 
 
+def are_counts_even(env):
```

Comment:
Same function naming and `namedtuple` issues as I mentioned in another PR recently.